src/components/ItemList/index.js

```diff
@@ -80,6 +80,38 @@ export function ItemList({ items }) {
         alert(err)
       })
   }
+
+  function AuthedBuyClick(itemIndex, authedflag) {
+    if (authedflag) {
+      return (
+        <Modal
+          aria-labelledby="simple-modal-title"
+          aria-describedby="simple-modal-description"
+          open={openPrice}
+          onClose={handleClickAwayPrice}
+        >
+          <div className={classes.paper}>
+            <h1>Excelent choice!</h1>
+            <h2>Owner Name </h2> <p>{itemIndex.ownerName}</p>
```

Comment:
`itemIndex` doesn't have the item properties. It seems it is just an index.
We need to get the item object.